### PR TITLE
Refactor AttendanceActivity.java

### PR DESCRIPTION
**Pull request by Redarcher9**

### CODE REVIEW

File: app/src/main/java/app/abhijit/iter/AttendanceActivity.java [hunk 1]
```diff
@@ -70,38 +70,38 @@
...
-    private Cache mCache;
-    private Student mNewStudent;
-    private Student mOldStudent;
-    private boolean mPrefExtendedStats;
-    private int mPrefMinimumAttendance;
-    private ArrayList<SubjectView> mSubjectViews;
-    private SubjectAdapter mSubjectAdapter;
+    private Context context;
+    private SharedPreferences sharedPreferences;
+    private Cache Cache;
```
Comment (abhijitparida) [inline app/src/main/java/app/abhijit/iter/AttendanceActivity.java]: Variable names must be in lowerCamelCase. The first character must be in lowercase.

**Comment by Redarcher9** [comment]
I sent another commit..Please check it out.

**Comment by abhijitparida** [comment]
Everything looks fine. Thank you!

**Comment by Redarcher9** [comment]
Thanks.. :+1:

**Comment by abhijitparida** [comment]
Issue #12 fixed

**Pull request merged**